pkg/service/aws/session.go

```diff
@@ -41,7 +41,32 @@ func NewSession(ctx context.Context, region string, accountName string) (*Sessio
 	conf := config.Get()
 	if conf.Env == "local" {
 		log.Println("running in dev mode, using ", conf.AWSAccessID)
-		awsCreds = credentials.NewStaticCredentials(conf.AWSAccessID, conf.AWSSecretKey, conf.AWSToken)
+
+		if conf.AWSAccessID != "" {
+			// creating creds from config variables
+			awsCreds = credentials.NewStaticCredentials(conf.AWSAccessID, conf.AWSSecretKey, conf.AWSToken)
+
+		} else {
+
+			log.Println("AWS_ACCESS_ID is empty, using 'default' profile from user's home directory")
```

Comment:
wrap it in a different function may be ?

Reply:
i think we can wrap while config read when env is local to different, lets avoid nested ifs